integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionTest.java

```diff
@@ -77,7 +77,7 @@
   @BeforeMethod
   public void setup() throws Exception
   {
-    // Set comapction slot to 10
+    // Set comapction slot to 5
```

Comment:
This comment was stale. The compaction slot was always 5. It was not changed in this PR.